src/Service/UrlBundleService.php

```diff
@@ -18,25 +19,19 @@ class UrlBundleService implements UrlBundleInterface
 {
     use Extensible;
     use Injectable;
+    use Configurable;
+
+    private static bool $strip_stage_param = false;
```

Comment:
@emteknetnz actually I'll ask the question on this one.

At the moment, the module behaviour could result in you getting links with `?stage=Stage` (IE any time this process runs and the reading mode just happens to be `DRAFT`), which is actually dangerous if it ever does happen.

With that in mind, what are your thoughts on this being a "breaking change" / security fix?

Reply:
Right, so there's no really legitimate reason to cache something with `?stage=Stage`

Instead of stripping `?stage=Stage`, seems like it would be better to just check what the current reading mode is and if it's DRAFT throw a hard exception?

Reply:
Hmmm. Actually, I'm gonna take my statement back. I suppose you **could** statically cache draft content and serve that when browsing `?stage=Stage`. I think that'd be an absolutely terrible author experience, but I can't go so far as to say that there is no legitimate reason for it.

Leaving the default as `false` to keep BC :)